roles/ora-host/tasks/main.yml

```diff
@@ -433,3 +431,28 @@
     - "{{ listdisks }}"
   when: asm_disk_management == "asmlib"
   tags: asm-disks
+  
+  - name: (asmlib) ASM listing results
```

Comment:
Was this intended to be permanent?  I usually do someting like ansible-playbook -vvvv when I need full command output.